### DISCUSSION TITLE
reportGroup for regex in v8

### DISCUSSION THREAD

**Discussion by danny-legit** [zricethezav/gitleaks#714] [Q&A] [answered]
Hey and congrats on v8!

One thing I've noticed that changed in this major version upgrade is the reportGroup parameter, which no longer exists. It was used to allow a specific capture group of the regex to be returned as the secret. I wanted to check if this is intentional or an unwanted regression. 
Should this be considered an unwanted regression, I would gladly open an issue/PR to resolve it

Thanks!!

**Comment by nirbashan17** [comment]
I encountered this issue as well, I need the new features of the version but I am currently using the groups feature.

**Comment by zricethezav** [comment]
>  Should this be considered an unwanted regression, I would gladly open an issue/PR to resolve it

@danny-legit 
I consider it an unwanted regression. I would happily accept a PR. There is an issue already tracking this https://github.com/zricethezav/gitleaks/issues/280

**Comment by zricethezav** [comment]
https://github.com/zricethezav/gitleaks/pull/734 @danny-legit

**Comment by zricethezav** [comment, marked as answer]
Use `secretGroup` in https://github.com/zricethezav/gitleaks/releases/tag/v8.1.0
